### DISCUSSION TITLE
Docker - How to run a Healthceck?

### DISCUSSION THREAD

**Discussion by sachinwadhwa** [AdguardTeam/AdGuardHome#2839] [General]
Adguard works fine, to auto-restart unstable containers, how can I implement a docker healthcheck?

**Comment by sachinwadhwa** [comment]
For now I am using it as  following, is there any better way?

    healthcheck:
      test: "/bin/netstat -pant | /bin/grep 53"
      interval: 45s
      timeout: 30s
      retries: 3

**Reply by ameshkov** [reply]
This one is not bad at all, it should always listen on that port.
